roles.md

```diff
@@ -18,24 +18,33 @@ The *Cultivator* is responsible for tending to the all hands meetings and import
    - facilitate discussion
    - keep time
 - review and merge notes (Weekly)
+- host and plan activity for social call (Monthly)
 - help member-owners stay organized (As needed):
   - add meetings and important dates to calendar after meeting
   - set up required loomio theads and decisions
   - send reminders on time-sensitive action items
-- run [OKR review and identification process](okrs.md#reviewing-and-identifying-okrs-🚧) (Quarterly)
 
 For more details, see our Guides for [scheduling][] and [hosting][]
 meetings (summarized above).
 
    [scheduling]: guides.md#scheduling-a-meeting
    [hosting]: guides.md#hosting-a-meeting
 
-**Length and Succession**:
-1 month. Last meeting of each month, the Cultivator asks for volunteers for the next person, if no one volunteers they can request those who have not been one recently to do so, which can be declined within based on discussion and with cause.
+**Authority:**
+- set agenda and determine discussion items for all hands
+
+**Limits:** (on Authority)
+- permission of member-workers needed to:
+	- cancel or reschedule call for week
+
+**Time Budget:** Up to 2 hours/week.
+
+**Length and Succession**: 1 month, rotates through member-workers. Last meeting of each month, the Cultivator asks for volunteers for the next person, if no one volunteers they can request those who have not been one recently to do so, which can be declined within based on discussion and with cause.
```

Comment:
maybe not the place to decide, but with succession, feels like something missing about knowing where to find the "big arc" of who's done a role. I feel like we lose that context when the ask is made for new ppl to step up. Might be nice to have one clear place, and I might like that that mentioned here if/once it exists.

But that's perhaps work for later, and no block.

Reply:
At this point that info would be contained in WGs history, but yeah, something to ponder. I think to address later tho